flowmachine/flowmachine/features/subscriber/label_event_score.py

```diff
@@ -0,0 +1,192 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# # License, v. 2.0. If a copy of the MPL was not distributed with this
+# # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from shapely.geometry import shape
+from shapely.geometry.base import BaseGeometry
+from typing import List, Dict, Union, Any
+
+from .scores import EventScore
+from .subscriber_location_cluster import _JoinedHartiganCluster
+from ...core import Query
+
+
+class LabelEventScore(Query):
+    """
+    Represents a label event score class.
+
+    This class will label a table containing scores based on a labelling
```

Comment:
What's the "output" of this labelling? Is it a new table, or will there be an added column to the existing table (or are the scores replaced with the labels)?

Reply:
Not sure if this is easy to do, but it would be great to have an example of the input and output in this docstring.